### DISCUSSION TITLE
Contao 5x

### DISCUSSION THREAD

**Discussion by Punktraum** [terminal42/contao-notification_center#301] [Q&A]
Hallo zusammen

Wird Notification Center demnächst für Contao 5x erhältlich sein?
Ich könnte das für ein aktuelles Projekt - welches ich auf Contao 5 aufgesetzt habe, ganz gut gebrauchen.

Besten Dank & Gruss
Andreas

**Comment by Toflar** [comment]
https://github.com/terminal42/contao-notification_center/discussions/277#discussioncomment-7632058
